### PR TITLE
Welcome hess

### PR DESCRIPTION
**Pull request by BHess2653**

### CODE REVIEW

**Comment by LevanceWam** [comment]
Looked over your code and everything looks ready to go

**Review by MikeSewell — APPROVED** [review]
Looks good to me!

File: bot/client.js [hunk 1]
```diff
@@ -57,4 +57,10 @@ client.on('message', (message) => {
   }
 });
 
+// This will send the welcome message for new users who join.
+client.on('guildMemberAdd', (member) => {
+  member.sendMessage(`${member.user.username}, Welcome to the Full Sail Armada! Enjoy your stay! and use !help to see list of commands!`);
```
Comment (reactivepixel) [inline bot/client.js:62]: Consider that your implementation solves the problem, and elegantly, but falls just short in that your solution ​within the client.js file does not implement the controller you created. So in effect, any further maintenance will require duplicate efforts in keeping both messages identical.

File: bot/controllers/welcome.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    // Aliasing 'this' as controller to allow for binding in actions
+    const controller = this;
+
+    // Array of all commands, see baseCommand.js for prototype
+    this.commands = [
+      new Command(
+        '!welcome',
+        '!welcome',
+        'Welcome Message',
+        'Replay welcome message recieved when you join the server.',
```
Comment (reactivepixel) [inline bot/controllers/welcome.js:18]: Spelling Error

**Review by reactivepixel — CHANGES_REQUESTED** [review]
Not ready to deploy as is. Spelling errors that are exposed to the user are not ideal.

Additionally, the controller issue will prevent this from being implemented in actuality.

**Pull request closed without merging**